src/ResourceManager/EventHub/Commands.EventHub/Commands.EventHub.csproj

```diff
@@ -86,16 +99,15 @@
     <None Include="..\AzureRM.EventHub.psd1">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="help\Test-AzureRmEventHubName.md" />
     <None Include="Microsoft.Azure.Commands.EventHub.format.ps1xml">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
     <None Include="MSSharedLibKey.snk" />
     <None Include="packages.config">
       <SubType>Designer</SubType>
     </None>
-    <None Include="StartupScripts\*.ps1">
-      <!-- <CopyToOutputDirectory>Always</CopyToOutputDirectory> -->
-    </None>
+    <None Include="StartupScripts\*.ps1" />
```

Comment:
Revert this to how it was previously - it will be necessary if any cmdlet aliases are added in the future.

Reply:
Resolved

Reply:
I still see the change, please revert.